sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/early_termination_policy.py

```diff
@@ -107,6 +140,13 @@ def __init__(
         delay_evaluation: int = 0,
         evaluation_interval: int = 1,
     ) -> None:
+        """Defines an early termination policy based on running averages of the primary metric of all runs.
```

Comment:
Should "running averages" be singular here? It sounds like it's a single average of one metric, assuming I'm reading this correctly.

Reply:
This is the summary that MFE provided for this class, but I agree with your reading. Sounds like singular fits best. I'll make the change.